docs/standard/microservices-architecture/container-docker-introduction/docker-containers-images-registries.md

```diff
@@ -9,9 +9,9 @@ ms.date: 05/26/2017
 
```

Comment:
Above, you should update the date since this is a major set of updates.

Reply:
Done.